R/dash.R

```diff
@@ -467,12 +469,16 @@ Dash <- R6::R6Class(
                           port = Sys.getenv('DASH_PORT', 8050), 
                           block = TRUE, 
                           showcase = FALSE, 
+                          pruned = TRUE, 
```

Comment:
Maybe a more verbose name for this flag? `pruned_errors` perhaps?

Reply:
Good idea, I'll make this change.

Reply:
fixed in 35dedac